File: SeelenkulinarikFront/src/app/card.ts
export class Card{
    public id: number;
    public Title: String;
    public Body: String;
    public AestheticURI: String;
    public ImageURI: String;

    [key: string]: any;

    public constructor(_id: number, _title: String, _body: String, _aest: String, _imageURI: String){
        this.id =_id;
        this.Title = _title;
        this.Body = _body;
        this.AestheticURI =_aest;
        this.ImageURI = _imageURI;
    }
}

File: Seelenkulinarik/src/main/java/at/Seelenkulinarik/Seelenkulinarik/Models/Card.java
package at.Seelenkulinarik.Seelenkulinarik.Models;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

@Entity
public class Card {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(nullable = false)
    public int id;

    public String Title;
    public String Body;
    public String AestheticURI;
    public String ImageURI;
}
